.github/workflows/ci.yml

```
@@ -58,6 +58,11 @@ jobs:
         contains(github.event.pull_request.labels.*.name, 'Run publish')
       )
     needs: [test]
-    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@main
+    with:
+      test_extras: 'dev'
+      test_command: 'pytest -p no:warnings --doctest-rst --pyargs fiasco'
```

```suggestion
      test_command: 'python -m pytest -p no:warnings --doctest-rst --pyargs fiasco'
```
I wonder your error is because you're picking up `pytest` from outside the virtual environment created to test the sdist? This might help (but also might not, a bit of a stab in the dark)